### PR TITLE
feat: create grid for mobile, tablet and desktop

### LINKED ISSUE #2: feat: create grid for mobile, tablet and desktop

### PR DESCRIPTION
**Pull request by Nicolleta1981**
fix #2

### CODE REVIEW

File: src/nicolleta1981/index.pug [hunk 1]
```diff
@@ -0,0 +1,16 @@
+extends ../_component/layout/base
+
+block awesome-content 
+  .grid 
+    .square(jumatate)
```
Comment (marianzburlea) [inline src/nicolleta1981/index.pug]: remove old code

**Review by Constantin-Matei — APPROVED** [review]
💯 :)

**Review by marianzburlea — APPROVED** [review]
mega bravo 💯

**Review by Adrian01C — APPROVED** [review]
well done hehe

**Pull request merged**